docs/mkdocs/detailed-command-usage/Clear-Temp.md

```diff
@@ -4,16 +4,18 @@
 <br>
 
 ## Overview
-Deletes temporary cached manifests stored in the `/Cache` directory.
-These files are cached in order to dramatically speed up future `prefill` runs (in some cases 3X faster).
-For most users it isn't necessary to use this command, however it may be useful to free up some disk space if you are running low on storage.
+Deletes temporary data created by {{prefill_name}}, such as saved manifests, to free up disk space.
+
+These files are saved locally in order to dramatically speed up future `prefill` runs (in some cases 3X faster), at the expense of increase disk usage.  These cached manifests will also build up over time, as newer versions of games are released, leaving unused manifests behind that will never be used again.
```

Comment:
Leaving cached in here as it's specifically mentioning manifests and therefore would not refer to lancache?

Reply:
This was actually an oversight from me.  I'm going to drop the word 'cached' here, it will now read `These manifests will also build up over time...` which I think is still clear.